src/Corax/Querying/IndexSearcher.Search.cs

```diff
@@ -214,6 +214,10 @@ private IQueryMatch SearchQueryWithPhraseQuery(FieldMetadata field, IEnumerable<
                     termMatches ??= new();
                     terms.Clear(); // Clear the terms list.
                     EncodeAndApplyAnalyzerForMultipleTerms(field, word, ref terms);
+
+                    //When single term outputs multiple terms we've to jump into phraseQuery
+                    if (terms.Count > 1)
+                        goto PhraseQuery;
```

Comment:
Just to make sure. This is how we behave in Lucene, right?

Reply:
Yes, it's not explicitly, but:
https://github.com/ravendb/ravendb/blob/8e6535d7353a94b60375f263cfb2ab24c009d02f/src/Raven.Server/Documents/Queries/LuceneQueryBuilder.cs#L899-L906
`GetValues()` will return this as single string (for sequences like `din%ner`), and then during `AnalyzedTerm` we will create `PhraseQuery` from it (since analyzer returns more than 1 term):
https://github.com/ravendb/ravendb/blob/8e6535d7353a94b60375f263cfb2ab24c009d02f/src/Raven.Server/Documents/Queries/LuceneQueryHelper.cs#L227-L243